lively.morphic/tests/derivationID-test.js

```diff
@@ -0,0 +1,29 @@
+/* global describe, it, beforeEach, afterEach */
+import { expect } from 'mocha-es6';
+import { Morph } from 'lively.morphic';
+
+describe('lively.merger >> Differ', () => {
```

Comment:
This is not in the module lively.merger>>differ, right? Maybe it's enough when the 'derivationId' is the top-level describe